bigchaindb/commands/bigchaindb.py

```diff
@@ -154,6 +156,23 @@ def run_init(args):
         print('If you wish to re-initialize it, first drop it.', file=sys.stderr)
 
 
+@configure_bigchaindb
+def run_recover(args):
+    _run_recover(BigchainDB())
+
+
+def _run_recover(b):
+    query.delete_zombie_transactions(b.connection)
+
+    tendermint_height = b.get_latest_block_height_from_tendermint()
+    block = b.get_latest_block()
+
+    if block:
+        while block['height'] > tendermint_height:
+            query.delete_latest_block(b.connection)
```

Comment:
Would you mind adding a `log.info` here, saying something like `bigchaindb is ahead of tendermint, removing block %s`